tests/functional/conftest.py

```diff
@@ -266,3 +267,9 @@ def project(
         # the following feels kind of fragile. TODO: better way of getting database
         database=profiles_yml['test']['outputs']['default']['dbname'],
     )
+
+
+@pytest.fixture
+def create_tables(data_dir, unique_schema):
```

Comment:
Putting a 'create_tables' fixture into here is fine for now. Jeremy was suggesting that long-term we might want to look at using seeds for this, but that's a bunch more conversion work, so I'd wait until we see how many different files like this we have.